gui/include/gui/RoutePropDlgImpl.h

```diff
@@ -112,24 +112,44 @@ class RoutePropDlgImpl : public RoutePropDlg {
   RoutePoint* m_pEnroutePoint;
   bool m_bStartNow;
 
-  // The timezone to use for date/time display and setting the departure time.
-  // 0 = UTC
-  // 1 = Timezone configured in operating system.
-  // 2 = Mean solar time at the location, based on the average time it takes for the sun
-  //     to cross the meridian (appear at its highest point in the sky) at that specific location.
-  // 3 = Honor OpenCPN global setting for timezone display.
+  /**
+   * @brief The timezone to use for formatting the departure date/time.
+   *
+   * @details The possible values are:
+   * - 0: UTC
+   * - 1: Timezone configured in operating system
+   * - 2: Mean solar time at the location, based on the average time it takes for the sun
+   *      to cross the meridian (appear at its highest point in the sky) at that specific location
+   * - 3: Honor OpenCPN global setting for timezone display
+   */
   int m_tz_selection;
 
   wxDataViewColumn* etd_col;
 
   wxHyperlinkCtrl* m_pEditedLink;
 
   bool IsThisRouteExtendable();
-  // Convert the date/time in UTC to the given format (UTC, Local@PC, LMT@Location).
+
+  /**
+   * @brief Convert the date/time in UTC to the given format.
```

Comment:
This is fine, and no need to change anything.

However, we use the javadoc conventions so the @brief and @details are not really needed. Without them the  first sentence  becomes the @brief part and the possible rest is  details.

You can see examples in _model/include/model/rest_server.h_ and how it is rendered in http://opencpn.github.io/OpenCPN/api-docs/classRestServer.html

Reply:
no worries, it's good to have consistency. I've made the changes.